src/main/java/com/lalabrand/ecommerce/item/category/CategoryController.java

```diff
@@ -0,0 +1,16 @@
+package com.lalabrand.ecommerce.item.category;
+
+import org.springframework.graphql.data.method.annotation.QueryMapping;
+import org.springframework.stereotype.Controller;
+import java.util.List;
+@Controller
+public class CategoryController {
+    private CategoryRepository categoryRepository;
+    public CategoryController(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+    @QueryMapping(name = "allCategories")
+    public List<Category> allCategories(){
```

Comment:
It won't work. In the category entity we have items with FetchType.LAZY.

Reply:
I don't know how to fix this correctly